errgroup/LICENSE.google

```diff
@@ -0,0 +1,24 @@
+Portions Copyright 2009 The Go Authors.
```

Comment:
Where did you find this?

Reply:
https://go.googlesource.com/sync/+/refs/tags/v0.11.0/LICENSE